clients/vscode-hlasmplugin/prepare_release.sh

```diff
@@ -0,0 +1,13 @@
+set -e
+set -x
+VERSION=$1
+CURRENT_BRANCH=$2
+RELEASE_NOTES=$3
+OLD_VERSION=$4
+DATE=$5
+
+{ printf "%s" "$RELEASE_NOTES"; cat ../../CHANGELOG.md; } > tmp.md
```

Comment:
```suggestion
{ echo -n $RELEASE_NOTES; cat ../../CHANGELOG.md; } > tmp.md
```
?

Reply:
That is actually not possible, think about newlines and various special characters that may end up in release notes...